Classes/Util/PBTerminalUtil.m

```diff
@@ -7,23 +7,36 @@
 
 #import "PBTerminalUtil.h"
 #import "Terminal.h"
+#import "iTerm2GeneratedScriptingBridge.h"
```

Comment:
Just call it `iTerm2.h`.

Reply:
This was just copied from their repo - are you sure you want me to rename it?